ios/keyman/Keyman/Keyman/SetUpViewController/SetUpViewController.swift

```diff
@@ -74,7 +72,7 @@ class SetUpViewController: UIViewController, UIWebViewDelegate {
   private func loadFromServer() {
     let appVersion = Version.current.majorMinor
     let url = "https://help.keyman.com/products/iphone-and-ipad/\(appVersion.plainString)"
-      + "/start/installing-system-keyboard"
+      + "/start/installing-system-keyboard?embed=ios"
```

Comment:
I wonder if you wanted to set the help site based on the tier
alpha, beta -> help.keyman-staging
stable -> help.keyman.com ?

Reply:
I'd love to see a centralised location for URLs. And then a centralised location for the host names so a switch is easy as pie.

Reply:
I mean... that actually exists.  https://github.com/keymanapp/keyman/blob/a55b084ad5e2971b27eb1bfbe0882fc3354c64aa/ios/engine/KMEI/KeymanEngine/Classes/KeymanHosts.swift#L64-L69

It's half that I forgot about it when writing this PR and half that making such a change _is_ more substantial... and since this PR is meant for 🍒-picking, I wanted to keep the changeset as simple as possible.

Reply:
Newest commit fixes this.